### DISCUSSION TITLE
[General] Status of Zinc language

### DISCUSSION THREAD

**Discussion by microbecode** [zkSync-Community-Hub/zksync-developers#789] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

What is the status of the Zinc ZK DSL? The repo is archived, but says it will be released "soon". https://github.com/matter-labs/zinc

**Comment by alhenaai** [comment]
The Zinc ZK DSL repository being archived and the message about its release "soon" suggests that the project might be undergoing significant changes or reevaluation. Since the repository is archived, it typically means that there are no active developments or updates being pushed to it at the moment.

For the latest updates or changes regarding the Zinc ZK DSL, I recommend checking the official ZKsync GitHub repositories or the ZKsync developers discussions on GitHub. This platform will provide more detailed and up-to-date information about the status of various projects, including Zinc.

You can visit the ZKsync developers discussions here: [ZKsync Developers Discussions on GitHub](https://github.com/zkSync-Community-Hub/zksync-developers/discussions).

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by kaymomin** [comment, marked as answer]
@microbecode The Zinc project has been discontinued for about 4 years, and the current repo status doesn’t reflect that. 

We’ll update the repository soon to ensure it accurately communicates the state of the project. Thanks for pointing this out!
